CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+## 3.1.0 (2019-01-08)
```

Comment:
Lets move the additions to master, there are a few other changes I believe that we will want to incorporate into a release.